controllers/user.js

```diff
@@ -35,6 +36,35 @@ async function add(req, res) {
 async function verify(req, res) {
   const verifiedEmailHash = await DB.verifyEmailHash(req.query.token);
 
+  const unsafeBreachesForEmail = await HIBP.getUnsafeBreachesForEmail(
+    sha1(verifiedEmailHash.email),
+    req.app.locals.breaches
+  );
+
+  if(unsafeBreachesForEmail) {
+    unsafeBreachesForEmail.forEach((breach) => {
+      breach.BreachDate = new Date(breach.BreachDate).toLocaleString("en-US", {year: "numeric", month: "long", day: "numeric"});
+      breach.DataClasses = breach.DataClasses.join(", ");
+    });
+  }
+
+  const unsubscribeUrl = EmailUtils.unsubscribeUrl(verifiedEmailHash);
+  const serverUrl = req.app.locals.SERVER_URL;
+
+  await EmailUtils.sendEmail(
+    verifiedEmailHash.email,
+    "Your Firefox Monitor report",
+    "report",
+    {
+      email: verifiedEmailHash.email,
+      date: new Date().toLocaleString("en-US", {year: "numeric", month: "long", day: "numeric"}),
```

Comment:
Not sure if we need some helper function for the date formatting since it's pretty verbose and duplicated here and above on L46.

We already have the code duplicated in the [`prettyDate`](https://github.com/mozilla/blurts-server/blob/f5e4bc8f50961bf7ce9abb047327bebb67593d91/hbs-helpers.js#L6-L9) HBS helper, so it'd be one more layer I guess.

Reply:
👍 a follow-up refactor is to make the same HBS helpers work across the website & email templates. But there's some funkiness with the way we've wrapped the helpers up in a custom `HBSHelpers` object.